server/src/visitors/YmlBaseVisitor.ts

```diff
@@ -1,20 +1,242 @@
 // tslint:disable-next-line: no-submodule-imports
+import { ParserRuleContext } from 'antlr4ts';
 import { AbstractParseTreeVisitor } from 'antlr4ts/tree/AbstractParseTreeVisitor';
+import { Range, TextDocument, TextEdit } from 'vscode-languageserver';
 
 import { YmlCompletionItemsProvider } from '../completion/YmlCompletionItemsProvider';
 import { YmlDefinitionProvider } from '../definitions';
-import { YmlParserVisitor } from '../grammar';
+import {
+    ActionBlockContext,
+    CombinedComparisonContext,
+    ComparisonContext,
+    Instruction_assignmentContext,
+    Instruction_ifContext,
+    InstructionContext,
+    YmlParserVisitor,
+} from '../grammar';
+import { IDocumentFormatSettings } from '../settings/Settings';
 
 export class YmlBaseVisitor extends AbstractParseTreeVisitor<void> implements YmlParserVisitor<void> {
     constructor(
         public completionProvider: YmlCompletionItemsProvider,
         public uri: string,
         public definitions?: YmlDefinitionProvider,
+        public docFormatSettings?: IDocumentFormatSettings,
+        public filePossibleEdits?: TextEdit[],
+        public document?: TextDocument,
     ) {
         super();
     }
 
     public defaultResult() {
         // no default
     }
+
+    public visitCombinedComparison(node: CombinedComparisonContext): void {
```

Comment:
What are combined comparisons?

Reply:
`condition && condition`, or `condition || condition`

Reply:
But “condition” ≠ “comparison”, hence my confusion. This thing sounds awfully specific, as if it were only for “4 < 5” and not for “isPlop() == true”, for example.

Reply:
I guess I'll have to take a deeper look at the name of the rules used in the grammar. I'll to it later.

Reply:
Did you take a proper note of https://github.com/yseop/vscode-yseopml/pull/172/files#r468649523?

Reply:
Done here: https://yseop-knowledge.atlassian.net/browse/VSC-6

Reply:
In the end, I did a commit to fix this in the PR. :smile: